pandas/io/json/json.py

```diff
@@ -547,6 +547,7 @@ def _get_object_parser(self, json):
 
         if typ == 'series' or obj is None:
             if not isinstance(dtype, bool):
+                # TODO: dtype is unused. Should this be an update on kwargs?
                 dtype = dict(data=dtype)
```

Comment:
If we can find an example that breaks because of this unused variable, that would be good for another PR.

Reply:
FYI, comment applies to many of the other variables in which you questioned whether it actually should be used, but previous authors didn't get around to implementing functionality with it.